src/test/java/uk/co/ramp/covid/simulation/place/RestaurantTest.java

```diff
@@ -29,8 +33,8 @@ public void initialise() throws JsonParseException, IOException {
         restaurant = new Restaurant(CommunalPlace.Size.MED);
         p1 = new Adult(30, Person.Sex.MALE);
         p2 = new Pensioner(67, Person.Sex.FEMALE);
-        Household h1 = new Household(Household.HouseholdType.ADULT, null);
-        Household h2 = new Household(Household.HouseholdType.PENSIONER, null);
+        Household h1 = new SmallFamily(null);
```

Comment:
Change these household types to SingleAdult. 
Are there checks to ensure each household has the correct number and types of inhabitants?